Neos.Flow/Classes/Cli/RequestBuilder.php

```diff
@@ -118,6 +118,14 @@ public function build($commandLine): Request
         $request = new Request();
         $request->setControllerObjectName(HelpCommandController::class);
 
+        // @TODO: Can this be implemented even better/cleaner?
+        if (in_array('--help', $commandLine)) {
```

Comment:
Tests are failing, because, `$commandLine` could be a string. It's not guaranteed to be an array.